### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by Natalyl3**
Updated index.md and the Classification section:

1. added extra information to the models descriptions;
2. added descriptions for parameters;
3. fixed error and typos in descriptions.

### CODE REVIEW

File: crowdkit/aggregation/classification/glad.py [hunk 1]
```diff
@@ -61,14 +63,14 @@ class GLAD(BaseClassificationAggregator):
...
     Args:
-        max_iter: Maximum number of EM iterations.
-        eps: Threshold for convergence criterion.
-        silent: If false, show progress bar.
-        labels_priors: Prior label probabilities.
-        alphas_priors_mean: Prior mean value of alpha parameters.
-        betas_priors_mean: Prior mean value of beta parameters.
-        m_step_max_iter: Maximum number of iterations of conjugate gradient method in M-step.
-        m_step_tol: Tol parameter of conjugate gradient method in M-step.
+        n_iter: The number of EM iterations.
```
Comment (pilot7747) [inline crowdkit/aggregation/classification/glad.py:66]: No, it's the maximum number

Comment (pilot7747) [inline crowdkit/aggregation/classification/glad.py:66]: No, it's the maximum number of iterations

File: crowdkit/aggregation/classification/glad.py [hunk 2]
```diff
@@ -23,13 +23,12 @@
 
 @attr.s
 class GLAD(BaseClassificationAggregator):
-    r"""Generative model of Labels, Abilities, and Difficulties.
+    r"""The $\boldsymbol{GLAD}$ (Generative model of Labels, Abilities, and Difficulties) model is a probabilistic model that parametrizes the abilities of workers and the difficulty of tasks.
```
Comment (pilot7747) [inline crowdkit/aggregation/classification/glad.py:26]: What's the difference between just` **bold**` and `$\boldsymbol{bold}$`?

**Review by pilot7747 — CHANGES_REQUESTED** [review]

**Comment by pilot7747** [comment]
Thanks for a good PR! Please fix failing tests, resolve merge conflicts, and take into account small comments above.

File: crowdkit/aggregation/classification/glad.py (hunk 1 above)
Comment (Natalyl3) [reply crowdkit/aggregation/classification/glad.py]: fixed

File: crowdkit/aggregation/classification/glad.py (hunk 2 above)
Comment (Natalyl3) [reply crowdkit/aggregation/classification/glad.py]: As I understood, the `$\boldsymbol{bold}$` is used in formulas. In other cases, the `**bold**` is needed. I fixed it in all files

**Review by pilot7747 — APPROVED** [review]

**Pull request merged**